components/authz-service/server/v2/authz.go

```diff
@@ -331,13 +346,14 @@ func (v *VersionSwitch) Interceptor(ctx context.Context,
 
 	// These methods skip the check, though they are in the relevant service
 	// definition:
-	switch info.FullMethod {
-	case "/chef.automate.domain.authz.Authorization/GetVersion":
+	fullMethod := info.FullMethod
+	if strings.HasSuffix(fullMethod, "GetVersion") ||
+		strings.HasSuffix(fullMethod, "ValidateProjectAssignment") {
```

Comment:
Not sure why we are adding this? I would think that we ValidateProjectAssignment should only ever be called in v2 mode? I might not be following here.

Reply:
This is the token problem we were seeing-- we call `ValidateProjAssignment` in the tokens create code regardless of version. A comment would be helpful (or we make tokens version aware)...